init.lua

```diff
@@ -32,6 +33,7 @@ minetest.register_tool("orbs_of_time:orb_night",{
 	inventory_image = "orbs_orb_night.png",
 	wield_image = "orbs_orb_night_weild.png",
 	stack_max=1,
+	groups = { tool=1 },
```

Comment:
Why is this group required? `register_tool` already makes it obvious that it's a tool.